docs/source/scripts/scheduling.py

```diff
@@ -60,7 +60,7 @@ def dense(width, height):
     for n in x:
         dsk, keys = trivial(int(n), 5)
         start = time()
-        get(dsk, keys)
+        get(dsk, keys)  # type: ignore
```

Comment:
Sigh, another weird one from `mypy`. Do you understand this? Pyright seems okay with it

Reply:
I suspect that
- the functions in that list are not typed and/or
- the functions in that list don't have the exact same signature

Reply:
Yeah, for sure it's related to having untyped signatures in the above functions, but my understanding was that `mypy` will typically infer `Any` for such a case, and instead I'm seeing `cannot call function of unknown type`. But not worth going into, I just thought it was weird.